repairnator/kubernetes-support/README.md

```diff
@@ -0,0 +1,131 @@
+# Deploy a high avaliable Repairnator in Kubernetes
+
+![Design](K8sRepairnatorDesign.jpg)
+
+
+## Setup mongodb and ActiveMQ
```

Comment:
Add a section `Prerequisites` to explain what must be installed before starting.

Reply:
This issue is addressed in the recent commits.